project_event/models/project_task.py

```diff
@@ -378,14 +378,26 @@ def get_message_body(self, action):
         return switcher.get(action)
 
     def get_message(self, action):
+        message = '<br>'
+        responsible = self.responsible_id.id
```

Comment:
@eilst set  responsible = self.responsible_id.id after checking  if self.activity_task_type == 'activity': 
other wise  responsible = none or ""

Reply:
Well, because I re-assign it if  self.activity_task_type == 'task':, and one of them is required, but you are rit=ght I ll put it after

Reply:
Done